### PR TITLE
RELATED: RAIL-3989 Update function list for access with latest

### PR DESCRIPTION
**Pull request by plavemar**
- remove getInstance function as it is not available
- add getDashboardDispatchForDashboard and getDashboardSelectForDashboard

JIRA: RAIL-3989

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/975/ : SUCCESS in 1m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/649/ : SUCCESS in 6s

File: docs/18_dashboard_component.md [hunk 1]
```diff
@@ -193,8 +192,9 @@ To access the state of the Dashboard component, use the [SingleDashboardStoreAcc
 
 To access more than one dashboard `store` object, use the [DashboardStoreAccessorRepository](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/model/store/storeAccessors/DashboardStoreAccessorRepository.ts) class. This class uses a map where the value is [DashboardStoreAccessor](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/model/store/storeAccessors/DashboardStoreAccessor.ts) for a single dashboard and the key is the dashboard's `ObjRef`.
 
-* `getInstance()` returns an instance of the `DashboardStoreAccessorRepository` object.
 * `getAccessorsForDashboard(dashboard)` returns an accessor object for a specific dashboard.
+* `getDashboardSelectForDashboard(dashboard)` returns a dashboardSelect object for a specific dashboard.
```
Comment (no23reason) [inline docs/18_dashboard_component.md:196]: Should be dashboardSelect in backticks. Also the following line ahs this issue. Also please make sure the naming in getDashboardSelect and getDashboardDispatch a few lines above is consistent (they should also use `dashboardDispatch` and `dashboardSelect` for the results' names)

**Comment by evgeniabogranskaya** [comment]
LGMT

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR490

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/977/ : SUCCESS in 1m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/195/ : SUCCESS in 44s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/90/ : SUCCESS in 38s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/285/ : SUCCESS in 19s

**Pull request merged**